lib/firebase.dart

```diff
@@ -67,3 +72,10 @@ Future<List<model.Person>> readPeople(String filter, String option) async {
 
   return peopleList.map((v) => model.Person.fromFirebaseMap(v)).toList();
 }
+
+Future<List<model.Message>> readMessages(String id) async {
```

Comment:
What's the id of? Consider expanding the var name a little

Reply:
Fixed